public/app/ui/components/search/header-search.html

```diff
@@ -1,7 +1,7 @@
 <div class="search header-search">
     <form data-bind="submit: submit">
         <div class="input-group">
-            <input type="text" name="query" placeholder="Search the collections&hellip;" class="form-control" data-bind="textInput: searchText" />
+            <input type="text" name="query" placeholder="Search the collections&hellip;" class="form-control pull-right" data-bind="textInput: searchText" />
```

Comment:
And Safari too (comment not super revealing as they are both webkit but I wanted to contribute something)